piet-direct2d/src/text.rs

```diff
@@ -24,6 +28,17 @@ use crate::dwrite::{self, TextFormat};
 pub struct D2DText {
     dwrite: DwriteFactory,
     device: d2d::DeviceContext,
+    loaded_fonts: Rc<RefCell<LoadedFonts>>,
```

Comment:
I have a feeling we might want to change this to Arc at some point, to allow resource loading across threads, but there's probably a bunch of stuff that's going to change to go fully multithreaded, and the compiler will tell us what to do.

Reply:
I'm specifically unsure about the thread safety of the contained types, and wanted to be defensive for the time being.